app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -207,5 +216,24 @@ def has_multiple_sites?(application_choice)
     def has_multiple_courses?(application_choice)
       Course.current_cycle.where(provider: application_choice.provider).many?
     end
+
+    def application_choices_with_accepted_states
+      @application_form
+        .application_choices
+        .includes(:course, :site, :provider, :offered_course_option, :interviews)
+        .order(id: :asc)
+        .reject { |ac| ApplicationStateChange::ACCEPTED_STATES.exclude?(ac.status.to_sym) }
```

Comment:
Would this be slightly easier to follow by reversing the double-negative?

```suggestion
        .select { |ac| ApplicationStateChange::ACCEPTED_STATES.include?(ac.status.to_sym) }
```

or even

```suggestion
        .select { |ac| ac.status.to_sym.in? ApplicationStateChange::ACCEPTED_STATES }
```

Reply:
Yep good call!